site/ja/guide/ragged_tensor.ipynb

```diff
@@ -0,0 +1,1633 @@
+        "    つまり`value_rowids[j]==row`のとき、\n",
+        "    `nrows`は`Ragged Tensor`の行の中で特定された一つの整数値\n",
+        "    を持ちます。　また`nrows`が使用される時は、空の行を示すときに使用されます。\n",
+        "\n",
+        "</dl>\n",
+        "\n",
+        "以下に例を示します。以下のRagged Tensorは全て同値にエンコードされます。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "4TH6XoQ8-PWh",
+        "colab": {}
+      },
+      "source": [
+        "values = [3, 1, 4, 1, 5, 9, 2, 6]\n",
+        "print(tf.RaggedTensor.from_row_splits(values, row_splits=[0, 4, 4, 7, 8, 8]))\n",
+        "print(tf.RaggedTensor.from_row_lengths(values, row_lengths=[4, 0, 3, 1, 0]))\n",
+        "print(tf.RaggedTensor.from_row_starts(values, row_starts=[0, 4, 4, 7, 8]))\n",
+        "print(tf.RaggedTensor.from_row_limits(values, row_limits=[4, 4, 7, 8, 8]))\n",
+        "print(tf.RaggedTensor.from_value_rowids(\n",
+        "    values, value_rowids=[0, 0, 0, 0, 2, 2, 2, 3], nrows=5))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ZGRrpwxjsOGr"
+      },
+      "source": [
+        "Ragged Tensorクラスは、それぞれの行分割ベクトルを生成するメソッドを定義しています。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "fIdn-hUBsoSj",
+        "colab": {}
+      },
+      "source": [
+        "rt = tf.ragged.constant([[3, 1, 4, 1], [], [5, 9, 2], [6], []])\n",
+        "print(\"      values: {}\".format(rt.values))\n",
+        "print(\"  row_splits: {}\".format(rt.row_splits))\n",
+        "print(\" row_lengths: {}\".format(rt.row_lengths()))\n",
+        "print(\"  row_starts: {}\".format(rt.row_starts()))\n",
+        "print(\"  row_limits: {}\".format(rt.row_limits()))\n",
+        "print(\"value_rowids: {}\".format(rt.value_rowids()))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "2r9XUpLUsdOa"
+      },
+      "source": [
+        "(注意：`tf.RaggedTensor.values` および`tf.RaggedTensors.row_splits`はクラスのプロパティ\n",
+        "として保持されますが、そのほかの行分割ベクトルを参照する際は、メソッドが呼び出されます。\n",
+        "これは`row_splits`は主としての行分割方法として表現できることを反映したもので\n",
+        "他の行分割ベクトルは呼び出されるごとに、生成しているということになります。)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "NBX15kEr-PWi"
+      },
+      "source": [
+        "それぞれの行分割スキーマの長所と短所をそれぞれ述べると、\n",
+        "\n",
+        "+ **効果的なインデックス指定をしたい場合**:\n",
+        "    `row_splits`や`row_starts`および`row_limits`スキーマは、\n",
+        "    一定時間でインデックス指定が可能です。また、`value_rowids`や\n",
+        "    `row_lengths`ではそれができません。\n",
+        "\n",
+        "+ **エンコードサイズを短くしたい場合**:\n",
+        "    `value_rowids`スキーマでは、空の行が多い場合に効果的にエンコードすることができます。\n",
```

Comment:
The translation for `since the size of the tensor depends only on the total number of values.` is missing.